tpgtools/overrides/monitoring/beta/metrics_scope.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Google LLC. All Rights Reserved.
```

Comment:
Same as GA- we can clean this up

Reply:
Done, and also removed GA overrides for monitored project since it is beta-only.